server/mongodb/models/Chapter.ts

```diff
@@ -31,6 +33,23 @@ const ChapterSchema = new Schema<Chapter>({
   },
 });
 
+ChapterSchema.pre(
+  "remove",
+  async function (next: (err?: CallbackError) => void) {
+    const id = this._id as Types.ObjectId;
+    // Cascade deletes (and remove chapter from projects)
+    await Promise.all([
+      // delete the user associated with the nonprofit
+      deleteDocumentAndDependencies(UserModel, {
+        chapter: id,
+      }),
```

Comment:
are you deleting the user that creates the nonprofit page, or the nonprofit alone?

Reply:
This deletes all users associated with the chapter (e.g., have the chapter field populated with this id)